### DISCUSSION TITLE
Clipboard sync with Android

### DISCUSSION THREAD

**Discussion by hollow-frenk** [oae/gnome-shell-pano#237] [Ideas]
Hi everyone, it would be great to be able to have a sort of continuity between the two Clipboard (mobile and desktop) maybe integrating pano into an Android keyboard a bit like it happens with Microsoft's swiftkey keyboard... What do you think?

**Comment by Totto16** [comment]
There is a cool Extension: [GSConnect](https://extensions.gnome.org/extension/1319/gsconnect/) that has these capabilities
